python/tank/platform/engine.py

```diff
@@ -636,6 +712,45 @@ def change_context(self, new_context):
     ##########################################################################################
     # public methods
 
+    def show_busy(self, title, details):
```

Comment:
unrelated code cleanup that accidentally went into this branch rather than the reshuffle branch... sorry!